aidants_connect_web/views/service.py

```diff
@@ -139,7 +139,17 @@ def recap(request):
         if form.get("personal_data") and form.get("brief"):
             mandat["aidant"] = user
             try:
-                usager.save()
+                usager, created = Usager.objects.update_or_create(
```

Comment:
I don't think we want to update the profile even though I would not be very risky but not very usefull. (The  user information from a sub will always be the same according to FC)

However, in the FC as FS, when we get the id_token, the sub is already in there.
We could check if we know the user there, and only call for the full profile if we don't know the user.